capa/ida/helpers.py

```diff
@@ -191,3 +199,53 @@ def read(self, size):
 
     def close(self):
         return
+
+
+def save_cached_results(resdoc):
+    logger.debug("saving cached capa results to netnode '%s'", CAPA_NETNODE)
+    n = netnode.Netnode(CAPA_NETNODE)
+    n[NETNODE_RESULTS] = resdoc.json()
```

Comment:
Does this create `CAPA_NETNODE` if it does not already exist? Is `n` always valid in this context?

Reply:
Yes, it creates a new one or accesses the existing one.